recipes/nghttp2/meta.yaml

```
@@ -0,0 +1,67 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "nghttp2" %}
+{% set version = "1.39.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # If getting the source from GitHub, remove the line above,
+  # uncomment the line below, and modify as needed.
+  url: https://github.com/nghttp2/{{ name }}/releases/download/v{{version}}/nghttp2-{{ version }}.tar.gz
+  sha256: 25b623cd04dc6a863ca3b34ed6247844effe1aa5458229590b3f56a6d53cd692
+  # sha256 is the preferred checksum -- you can get it for a file with:
+  #  `openssl sha256 <file name>`.
+  # You may need the openssl package, available on conda-forge:
+  #  `conda install openssl -c conda-forge``
+
+build:
+  # Uncomment the following line if the package is pure Python and the recipe is exactly the same for all platforms.
+  # It is okay if the dependencies are not built for all platforms/versions, although selectors are still not allowed.
+  number: 0
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  skip: True  # [not unix]
+
+requirements:
+  build:
+    # If your project compiles code (such as a C extension) then add the required compilers as separate entries here.
+    # Compilers are named 'c', 'cxx' and 'fortran'.
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - make
+    - pkg-config  # [unix]
```

Would add an empty `host` section below. This is useful for splitting the `build` and `host` environments and ensuring there is no overlap of files between the two.

```suggestion
    - pkg-config  # [unix]
  host:
```